lib/main.js

```diff
@@ -89,11 +89,11 @@ export default {
       requireResolve(TSLINT_MODULE_NAME, { basedir },
         (err, linterPath, pkg) => {
           let linter;
-          if (!err && pkg && /^3|4\./.test(pkg.version)) {
+          if (!err && pkg && /^3|4|5\./.test(pkg.version)) {
             if (pkg.version.startsWith('3')) {
               // eslint-disable-next-line import/no-dynamic-require
               linter = shim(require('loophole').allowUnsafeNewFunction(() => require(linterPath)));
-            } else if (pkg.version.startsWith('4')) {
+            } else if (pkg.version.startsWith('4') || pkg.version.startsWith('5')) {
```

Comment:
I agree, if a future v6 requires another change we can make this explicit again there.